src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanEnterpriseBuildStep.java

```diff
@@ -221,22 +221,22 @@ public void setTarget(String target) {
 	public String getTarget() {
 		return m_target;
 	}
-	
+
```

Comment:
The best thing to do is to commit only the feature in a particular commit, then to have a clean up commit afterward